zerver/tests/test_reactions.py

```diff
@@ -87,29 +87,35 @@ def test_cached_reaction_data(self) -> None:
         """
         Formatted reactions data is saved in cache.
         """
-        sender = self.example_user("hamlet")
-        reaction_info = {
-            "emoji_name": "smile",
-        }
-        result = self.api_post(sender, "/api/v1/messages/1/reactions", reaction_info)
+        senders = [self.example_user("hamlet"), self.example_user("cordelia")]
```

Comment:
Ideally I'd like to use this on the test above as well (`test_valid_emoji`) - but that test currently directly queries the Reaction database rather than using the API.

From zulip's testing philosophy, I'd like to switch the above test to use an `api_get` of `api/v1/messages` rather than directly querying the Reaction table.

Reply:
Yeah, feel free to do that in a second commit in this pull request.  (It's an independent but related change, which we generally like to handle with multiple commits in a single pull request, since we use a rebase-based merge strategy).